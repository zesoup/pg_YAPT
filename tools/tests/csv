$config = {
    tests     => 1,
    defaultui => 'default',

    UI => {
        default => {
            template   => "csv",
            updatetime => 10000,    #ns
            checks => [ "WAL", "SIZE", "UpTime", "TotRows", "S/I", "I/U/D" ]
          }

    }
};
