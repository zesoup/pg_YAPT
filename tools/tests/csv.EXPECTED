0|0MB;0GB;0h;0.0m;0/0|0k;0/0/0|0k
